CMakeLists.txt

```diff
@@ -409,23 +411,40 @@ if (NOT is_msvc)
   set(no_unused_w -Wno-unused-function)
 endif()
 
-add_with_props(rippled_src src/ripple/unity/snappy.cpp
+add_with_props(rippled_src_all src/ripple/unity/snappy.cpp
   -I"${CMAKE_SOURCE_DIR}/"src/snappy/snappy
   -I"${CMAKE_SOURCE_DIR}/"src/snappy/config
   ${no_unused_w})
 
 if (APPLE AND is_clang)
-  list(APPEND rippled_src src/ripple/unity/beastobjc.mm)
+  list(APPEND rippled_src_all src/ripple/unity/beastobjc.mm)
 endif()
 
+list(APPEND rippled_src_unity "${rippled_src_all}")
+list(APPEND rippled_src_nonunity "${rippled_src_all}")
+
 ############################################################
 
 if (WIN32 OR is_xcode)
   group_sources(src)
   group_sources(docs)
+  group_sources(Builds)
```

Comment:
How come Builds is being added?  It looks like it adds `CMakeFuncs.cmake` to the IDE?